### PR TITLE
Add MV2 timeline blog post & known issues doc

### PR DESCRIPTION
**Pull request by dotproto**

### CODE REVIEW

File: site/en/docs/extensions/whatsnew/index.md [hunk 1]
```diff
@@ -3,18 +3,29 @@ layout: 'layouts/doc-post.njk'
...
 <!--lint disable first-heading-level-->
 
 Check this page often to learn about changes to the Chrome extensions platform, its documentation,
 and related policy or other changes.
 
+### Blog post: More details on the transition to Manifest V3 {: #blog-mv3-transition-details }
+
+September 28, 2022
+
+[This blog post](/blog/more-mv2-transition/) provides clarifications on the Manifest V2 deprecation timeline. The [Manifest V2 support timeline](/docs/extensions/mv3/mv2-sunset/) has also been updated to reflect this information.
```
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:21]:
```suggestion
We have [clarified the Manifest V2 deprecation timeline](/blog/more-mv2-transition/). The [Manifest V2 support timeline](/docs/extensions/mv3/mv2-sunset/) has also been updated to reflect this information.
```

File: site/en/docs/extensions/whatsnew/index.md [hunk 2]
```diff
@@ -3,18 +3,29 @@ layout: 'layouts/doc-post.njk'
...
+
+### Docs update: Known issues when migrating to Manifest V3 {: #known-issues }
+
+September 28, 2022
+
+[This new article](/docs/extensions/mv3/intro/) provides an overview of major features currently in development and open bugs.
```
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:27]: This isn't a new article. Is this the right link?

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/en/docs/extensions/whatsnew/index.md (hunk 2 above)
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:27]: Oops, wrong page! I meant to link to the known issues page. 

```suggestion
[This new article](/docs/extensions/mv3/known-issues/) provides an overview of major features currently in development and open bugs.
```

File: site/en/docs/extensions/whatsnew/index.md [hunk 3]
```diff
@@ -3,18 +3,29 @@ layout: 'layouts/doc-post.njk'
...
+
+September 28, 2022
+
+We have [clarified the Manifest V2 deprecation timeline](/blog/more-mv2-transition/). The [Manifest V2 support timeline](/docs/extensions/mv3/mv2-sunset/) has also been updated to reflect this information.
+
+### Docs update: Known issues when migrating to Manifest V3 {: #known-issues }
+
+September 28, 2022
+
+[This new article](/docs/extensions/mv3/known-issues/) provides an overview of major features currently in development and open bugs.
```
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:27]:
```suggestion
We've put together a list of [major features currently in development and open bugs](/docs/extensions/mv3/known-issues/).
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/en/docs/extensions/whatsnew/index.md [hunk 4]
```diff
@@ -3,18 +3,29 @@ layout: 'layouts/doc-post.njk'
...
+
+September 28, 2022
+
+We have [clarified the Manifest V2 deprecation timeline](/blog/more-mv2-transition/). The [Manifest V2 support timeline](/docs/extensions/mv3/mv2-sunset/) has also been updated to reflect this information.
+
+### Docs update: Known issues when migrating to Manifest V3 {: #known-issues }
+
+September 28, 2022
+
+We've put together a list of [major features currently in development and open bugs](/docs/extensions/mv3/known-issues/).
```
Comment (dotproto) [inline site/en/docs/extensions/whatsnew/index.md:27]: ATM this feels a bit terse. Is there anything else that we could say? I was considering something along the lines of "Our goal with this page is to help developers better understand the current state of the platform and what features they can target as they prepare for the future", but I'm on the fence about it.

Comment (jpmedley) [reply site/en/docs/extensions/whatsnew/index.md:27]: SGTM

Comment (dotproto) [inline site/en/docs/extensions/whatsnew/index.md:27]:
```suggestion
We've put together a list of [major features currently in development and open bugs](/docs/extensions/mv3/known-issues/). Our goal with this page is to help developers better understand the current state of the platform and what features they can target as they prepare for the future.
```

**Review by jpmedley — APPROVED** [review]

**Pull request merged**